setup.cfg

```diff
@@ -13,12 +13,11 @@ classifiers =
     License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)
     Natural Language :: English
     Operating System :: OS Independent
-    Programming Language :: Python :: 3.6
-    Programming Language :: Python :: 3.7
```

Comment:
Is there any point in taking away 3.7 compatibility explicitly? If the tests still pass on 3.7 and someone just happends to have 3.7 installed, is there any reason to prevent it being used?

Reply:
They'll still get the last release that supported 3.7 from pypi, so they can keep using it if they want to. Those aren't going anywhere. This just clearly indicates that releases from here on won't support 3.7. 3.7 went EOL on 2023-06-27, so it's already been 5 months. By the time we make another release, it'll likely be more than 6. I think that's enough time for us to hold on for folks?